packages/styles/package.json

```diff
@@ -21,7 +21,8 @@
     "lint": "stylelint 'src/**/*.scss'",
     "serve": "serve -p 5000",
     "test:watch": "echo 'No tests available yet.'",
-    "test": "node src/test.js"
+    "test": "node src/test.js",
```

Comment:
I'd do this the other way around.

```
"test": "npm run test:styles",
"test:styles": "node src/test.js"
```

What do you think?

Reply:
+1

Reply:
That's fine for me. I started with that indeed, and then simplified it. Are we expecting more type of tests here?